### ISSUE TITLE
Transpose: Support ContinousVariable as Feature name

### ISSUE THREAD

**Issue by stuart-cls** [biolab/orange3#3677]
The Transpose widget should support naming features after ContinousVariables in metas.

##### Orange version
3.20.1

##### Expected behavior
Select ContinousVariable from metas, str(number) is set as Feature name in transposed table.

##### Actual behavior
"From meta attribute" remains greyed out, because:

````
DomainModel(
            order=DomainModel.METAS, valid_types=StringVariable,
            alphabetical=True)
````

**Comment by janezd** [comment]
Isn't such a variable then actually a string, and the user should change its type, say when loading?

**Comment by stuart-cls** [comment]
Ah, sorry perhaps I wasn't clear. The ContinousVariable is a float64, but the new `Variable.name` must be a string.

For context, in orange-spectroscopy we do:

    features = [Orange.data.ContinuousVariable.make("%f" % f) for f in domvals]

where `domvals` is an array of energies stored as floats.

[1] https://github.com/Quasars/orange-spectroscopy/blob/e0b902ca8cdfdd15b6e9400256a2412f77cfbdcb/orangecontrib/spectroscopy/data.py#L966

**Comment by janezd** [comment]
I understood what you meant, but I haven't realized that you actually have a perfectly valid use case.

To anybody who implements it: if values are not unique, add numbers to variables names (see if `Orange.data.util.get_unique_names` is useful for this), and also show a warning.

**Comment by janezd** [comment]
Closed via #3721.

**Comment by stuart-cls** [comment]
Cool, thanks!
